src/Microsoft.Dnx.TestHost/Project.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.Dnx.TestHost
+{
+    public class Project
+    {
+        private const string ProjectFileName = "project.json";
+
+        public string Name { get; set; }
+
+        public IDictionary<string, string> Commands { get; }
+            = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+
+        public string EntryPoint { get; set; }
```

Comment:
@davidfowl On looking further, looks like we cannot remove the Json.net dependency as we use it for serialization too(our own json parser only supports deserliazation) https://github.com/aspnet/Testing/blob/dev/src/Microsoft.Dnx.TestHost/ReportingChannel.cs#L68